packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -646,6 +720,140 @@ class OpenUpwardsPageTransitionsBuilder extends PageTransitionsBuilder {
   }
 }
 
+/// Used by [PageTransitionsTheme] to define a horizontal [MaterialPageRoute] page
+/// transition animation that looks like the default page transition
+/// used on Android U.
+///
+/// {@tool dartpad}
+/// This example shows the default page transition on Android.
+///
+/// ** See code in examples/api/lib/material/page_transitions_theme/page_transitions_theme.3.dart **
+/// {@end-tool}
+///
+/// See also:
+///
+///  * [FadeUpwardsPageTransitionsBuilder], which defines a page transition
+///    that's similar to the one provided by Android O.
+///  * [OpenUpwardsPageTransitionsBuilder], which defines a page transition
+///    that's similar to the one provided by Andoird P.
+///  * [ZoomPageTransitionsBuilder], which defines the default page transition
```

Comment:
If we are going to keep the old transition, have we thought about having android version as part of theme? for example

ThemeData.androidVersion, and then based on version we build different default pagetransition

Reply:
I actually think we should deprecate these old page transitions in the future, as they are already outdated:)